packages/harmony/.storybook/preview.ts

```diff
@@ -13,3 +16,15 @@ import 'assets/styles/shadows.css'
 export const parameters = {
   viewMode: 'docs'
 }
+
+export const decorators = [
+  withThemeByDataAttribute({
+    themes: {
+      default: 'default',
+      dark: 'dark',
+      matrix: 'matrix'
+    },
+    defaultTheme: 'light',
```

Comment:
`light` doesn't match any theme here since you named it `default: 'default'`

Reply:
🤦 ty